examples/preconditioned-solver/preconditioned-solver.cpp

```diff
@@ -86,6 +86,11 @@ int main(int argc, char* argv[])
 
     // executor where Ginkgo will perform the computation
     const auto exec = exec_map.at(executor_string)();  // throws if not valid
+    exec->add_logger(gko::log::create_nvtx_hook());
+    if (exec->get_master() != exec) {
+        // mark host operations in gray
+        exec->get_master()->add_logger(gko::log::create_nvtx_hook(0x555555U));
+    }
```

Comment:
Is this necessary here ? Maybe create a separate example to demonstrate the ProfilerHooks ?

Reply:
I forgot to remove all changes to this file, will be gone in a second :)